### PR TITLE
Pagination for tables and API

### PR DESCRIPTION
**Pull request by roosnic1**
Enables pagination on the list queries. Currently only on the user list query. Will be later implemented later for other list queries also.

⚠️ PR will be merged into `f/WEP-23-member-plans`

### CODE REVIEW

**Review by faebebin — APPROVED** [review]
Looks good. But I am not so deep into API yet ;)

File: packages/editor/src/client/routes/userList.tsx [hunk 1]
```diff
@@ -119,51 +138,90 @@ export function UserList() {
...
+            <Cell dataKey={'name'}>
+              {(rowData: FullUserFragment) => (
+                <Link route={UserEditRoute.create({id: rowData.id})}>
+                  {rowData.name || t('userList.overview.unknown')}
+                </Link>
+              )}
+            </Cell>
+          </Column>
+          <Column width={400} align="left" resizable>
+            <HeaderCell>{t('email')}</HeaderCell>
```
Comment (0815pascal) [inline packages/editor/src/client/routes/userList.tsx:181]: I would rather use here `userList.overview.email` in order to be consistent with the rest

File: packages/editor/src/client/routes/userList.tsx [hunk 2]
```diff
@@ -119,51 +138,90 @@ export function UserList() {
...
+            <Cell dataKey="email" />
+          </Column>
+          <Column width={100} align="center" fixed="right">
+            <HeaderCell>{t('action')}</HeaderCell>
```
Comment (0815pascal) [inline packages/editor/src/client/routes/userList.tsx:185]: I would rather use here `userList.overview.action` in order to be consistent with the rest

**Review by 0815pascal — CHANGES_REQUESTED** [review]
Looks good, just two tiny things and I noticed that you forgot to add the German and French translation files.

**Review by penk85 — APPROVED** [review]

**Review by faebebin — APPROVED** [review]
Looks good.

**Review by bishoymelek — CHANGES_REQUESTED** [review]

File: packages/api/src/db/user.ts [hunk 3]
```diff
@@ -58,6 +67,15 @@ export interface UserSubscriptionPeriod {
   readonly invoiceID: string
 }
 
+export interface UserSubscriptionPeriodInput {
```
[the remark this answers is not in the document]
Comment (roosnic1) [reply packages/api/src/db/user.ts:70]: It's the same but they might change in the future. One is to handle the input and the other is the whole representation.

**Pull request merged**